dwebsummit/dwebsummit_frontend/static/css/_desktop.scss

```diff
@@ -81,21 +81,19 @@
     margin-top: 0rem;
     margin-bottom: 2rem;
   }
-  section.sponsors {
+  #sponsors {
     .grid { padding-left:5px; }
-    .lead-sponsors-grid, .sponsors-grid {
-      .box { padding: 0 1rem; display: inline-block;}
-    }
-    .lead-sponsors-grid {
-      .sponsor-img {
-        height: 25rem;
-        width: auto;
+    section {
+      .sponsors-grid {
+        .box { padding: 0 1rem; display: inline-block;}
```

Comment:
Suggestion: Even though this is a copy paste, since you’re in it, I’d break up the attributes onto separate lines.